pack.sh

```diff
@@ -25,22 +25,31 @@ function lerna_scopes() {
   done
 }
 
-echo "Packaging jsii modules" >&2
+# Compile examples with respect to "decdk" directory, as all packages will
+# be symlinked there so they can all be included.
+echo "Extracting code samples" >&2
+node --experimental-worker $(which jsii-rosetta) \
+  --compile \
+  --output samples.tabl.json \
+  --directory packages/decdk \
+  $(cat $TMPDIR/jsii.txt)
```

Comment:
This should ideally be in the build step of each module so that I can see errors on invalid example code. If this happens only in the pack step, I won't know of this until a PR build. 
Why not do this in the build step as done in compile samples?

Also, this is technically not 'packing'. If I were to speed up PR builds by removing packing, we would still want this to be present.

Reply:
> Why not do this in the build step as done in compile samples?

Because it's not hermetic. Samples will require everything to have been built and linked into `decdk` package.